v2/azure-arm.yaml

```diff
@@ -938,6 +938,9 @@ objectModelConfiguration:
         $exportAs: Database
         $supportedFrom: v2.0.0-beta.1
   dbformysql:
+    v1: # Handcrafted 
```

Comment:
Do we have a way to detect if we've forgotten to add this?

Reply:
I'm not sure at the moment. I'll give that some thought though.